src/app/core/entity-details/entity-details/entity-details.component.ts

```diff
@@ -60,77 +59,79 @@ import { EntityArchivedInfoComponent } from "../entity-archived-info/entity-arch
     EntityArchivedInfoComponent,
   ],
 })
-export class EntityDetailsComponent {
-  entity: Entity;
+export class EntityDetailsComponent implements EntityDetailsConfig, OnChanges {
   creatingNew = false;
   isLoading = true;
 
-  panels: Panel[] = [];
-  config: EntityDetailsConfig;
+  /** @deprecated use "entityType" instead, this remains for config backwards compatibility */
+  @Input() set entity(v: string) {
+    this.entityType = v;
+  }
+  @Input() entityType: string;
   entityConstructor: EntityConstructor;
 
+  @Input() id: string;
+  record: Entity;
+
+  @Input() panels: Panel[] = [];
+  /** the actual, fully resolved panel configs */
+  panelsComponents: Panel[] = [];
```

Comment:
```suggestion
  panelComponents: Panel[] = [];
```

Reply:
And why does this now need to be in a different property? I think nobody will understand what the differences between these two is.

Reply:
I started writing a long comment here, explaining why this was the best I could come up with ... and then realized that apparently my other refactorings made this unnecessary now. :sweat_smile: 

Good find. Removed now.